M/MONA/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MONA"
+version = v"1.4.18"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://www.brics.dk/mona/download/mona-1.4-18.tar.gz",
+                  "ece10e1e257dcae48dd898ed3da48f550c6b590f8e5c5a6447d0f384ac040e4c"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/mona*
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+if [[ "${target}" == *-apple-* ]]; then
+  make -j$(nproc) LDFLAGS="-Xlinker -undefined -Xlinker dynamic_lookup"
```

Comment:
I wonder whether `LDFLAGS` is already set here. If so, you would need to write `LDFLAGS="${LDFLAGS} -Xlinker -undefined -Xlinker dynamic_lookup"` instead.

Reply:
For safety, I've added this to the tarbal